lms/static/coffee/spec/instructor_dashboard/send_email_spec.coffee

```diff
@@ -0,0 +1,82 @@
+describe "Bulk Email Queueing", ->
```

Comment:
IMO, this is few enough lines that it's reasonable that it should be rewritten to JS to keep with our previous decision to not add any new coffeescript files.

cc @andy-armstrong